ral-facilities/scigateway
# Feature/external strings #53

## Description
Added a "strings" file (path included in settings.json) to contain all UI strings. Updated the current application components to pull strings and messages from this file

## Testing instructions
Start application, no changes should be observed from previous build.
Edit the strings in the default.json file and verify these are reflected in the UI

- [x] Check build

## Agile board tracking
connect to #53
